src/main/java/spoon/reflect/declaration/CtTypeInformation.java

```diff
@@ -180,15 +203,16 @@ public interface CtTypeInformation {
 	Collection<CtExecutableReference<?>> getAllExecutables();
 
 	/**
-	 * @return the type erasure, which is computed by the java compiler to ensure that no new classes are created for parametrized types so that generics incur no runtime overhead.
-	 * See https://docs.oracle.com/javase/tutorial/java/generics/erasure.html
+	 * @return the type erasure, which is computed by the java compiler to ensure that no new classes are created
+	 *         for parametrized types so that generics incur no runtime overhead.
```

Comment:
The runtime overhead isn't quite the main reason? Maybe just delete that part and only say that the type erasure is returned?

Reply:
That would make the doc even less understandable. I tried explaining what this method does